src/control/common/storage/commands.go

```diff
@@ -36,6 +36,18 @@ const MsgStoragePrepareWarn = "Memory allocation goals for SCM will be changed a
 	"are not in use. Please be patient as it may take several minutes " +
 	"and subsequent reboot maybe required.\n"
 
+type cmdLogger interface {
```

Comment:
I don't think this interface needs to be here -- isn't referenced anywhere in this package as far as I can tell.

Reply:
this is removed in https://github.com/daos-stack/daos/pull/1075